app/bundles/LeadBundle/Form/Type/FieldType.php

```diff
@@ -37,14 +37,8 @@
 
 class FieldType extends AbstractType
 {
-    /**
```

Comment:
This annotations shouldn't be removed. Probably was removed on windows by cs-fixer (it's bug). But need get back

Reply:
Hi @kuzmany indeed its weird i saw that but I didn't remove it on purpose. I was planning to adding it again then.
anyway, i will fix it